pkg/artifactory/provider.go

```diff
@@ -174,17 +155,14 @@ func buildResty(URL string) (*resty.Client, error) {
 	return restyBase, nil
 }
 
-func addAuthToResty(client *resty.Client, username, password, apiKey, accessToken string) (*resty.Client, error) {
+func addAuthToResty(client *resty.Client, apiKey, accessToken string) (*resty.Client, error) {
 	if accessToken != "" {
 		return client.SetAuthToken(accessToken), nil
 	}
 	if apiKey != "" {
 		return client.SetHeader("X-JFrog-Art-Api", apiKey), nil
 	}
-	if username != "" && password != "" {
-		return client.SetBasicAuth(username, password), nil
-	}
-	return nil, fmt.Errorf("no authentication details supplied")
+	return nil, fmt.Errorf("no authentication details suppliedaccessToken := d.Get(\"access_token\").(string)")
```

Comment:
This message will mean nothing to a user. Not sure why you changed the previous message

Reply:
mistake! fixed